graylog2-web-interface/src/pages/ConfigurationsPage.tsx

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import { useEffect, useMemo, useState } from 'react';
+import { chunk } from 'lodash';
+import { ErrorBoundary } from 'react-error-boundary';
+import { SystemConfiguration } from 'views/types';
+
+import { Col, Row } from 'components/graylog';
+import { DocumentTitle, PageHeader, Spinner } from 'components/common';
+import { useStore } from 'stores/connect';
+import CombinedProvider from 'injection/CombinedProvider';
+import { isPermitted } from 'util/PermissionsMixin';
+import SearchesConfig from 'components/configurations/SearchesConfig';
+import MessageProcessorsConfig from 'components/configurations/MessageProcessorsConfig';
+import SidecarConfig from 'components/configurations/SidecarConfig';
+import EventsConfig from 'components/configurations/EventsConfig';
+import UrlWhiteListConfig from 'components/configurations/UrlWhiteListConfig';
+import {} from 'components/maps/configurations';
+import style from 'components/configurations/ConfigurationStyles.lazy.css';
```

Comment:
These styles should eventually be used through styled components but I think we should take that into another PR. Probably also improving some margins and styling, since the page needs some for a long time.